docs/how-to/crafting/enable-classic-confinement.rst

```diff
@@ -38,7 +38,8 @@ Request classic confinement on the Snap Store
 ---------------------------------------------
 
 A classically-confined snap requires approval from the Store team before you can
-distribute it on the Snap Store.
+distribute it on the Snap Store. Obtaining approval takes takes on average three to five
+days.
```

Comment:
It's best to prime users to the idea that the request turnaround is on the order of days.

Should I change it to "business days"?

Reply:
I think tso.

Reply:
3a4ac59